AppImageGenerator.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <UserSecretsId>9423d0e8-1eb9-474b-9ce8-a383ce07fab7</UserSecretsId>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+    <DockerfileContext>.</DockerfileContext>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.1" />
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.17.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="2.1.3" />
+    <PackageReference Include="SkiaSharp" Version="2.88.3" />
```

Comment:
If you're only using ImageSharp for outputing the files, SkiaSharp should be also able to do this with SKBitmap.Encode

https://learn.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/bitmaps/saving#exploring-the-image-formats - at least for Jpeg/Png/Webp - do you need BMP/TIFF output support? Is that the constraint?

Eitherway, should at least comment/document why using two graphics library. It looks like you're using SkiaSharp for the SVG loading/rendering for sure.

Reply:
ImageSharp is used for image mutation like resize and background fill for raster images. SkiaSharp for SVG manipulation.